### DISCUSSION TITLE
Can't apply Student Pack

### DISCUSSION THREAD

**Discussion by liabatlle** [community/community#17778] [GitHub Education]
Hello GitHub 

I cannot apply for Student Pack because I do not have a student card when taking a Professional Training, pre-university studies. Very often the emails of some schools do not include the .edu in the mail.

There are pages like ISIC that generate a student card but you have to pay every year you want to renew and not everyone can afford it.

Please help me, as many pages do not accept us.

I hope to represent more people who are going through the same thing.

Coordinated greetings

**Comment by mlarasusan** [comment]
Hello [liabatlle](https://github.com/liabatlle)👋

I have some information here that can help you [✨](https://emojipedia.org/sparkles/)

👉 Check [this](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/) to learn how to get the GitHub Student Developer Pack without a student ID
👉 Also, check the reviews of these two students: [here](https://github.com/orgs/github-community/discussions/31#discussioncomment-2561585) & [here](https://github.com/orgs/github-community/discussions/16)
👉 And here is the [link](https://support.github.com/contact/education) to contact our support team 
👉 Student Developer Pack Application & [FAQs](https://github.com/orgs/github-community/discussions/9)
